x-pack/plugins/infra/public/components/metrics_explorer/chart_context_menu.tsx

```diff
@@ -28,6 +30,27 @@ interface Props {
   timeRange: MetricsExplorerTimeOptions;
 }
 
+const fieldToNodeType = (
+  source: SourceQuery.Query['source']['configuration'],
+  field: string
+): InfraNodeType | undefined => {
+  if (source.fields.host === field) {
+    return InfraNodeType.host;
+  }
+  if (source.fields.pod === field) {
+    return InfraNodeType.pod;
+  }
+  if (source.fields.container === field) {
+    return InfraNodeType.container;
+  }
+};
+
+const dateMathToEpoch = (dateMath: string, roundUp = false): number => {
```

Comment:
When first reading this really wondered what the `dateMath` in the function name and the argument was. Would `dateMathExpression` be too long?

```suggestion
const dateMathExpressionToEpoch = (dateMathExpression: string, roundUp = false): number => {
```